go.mod

```diff
@@ -3,112 +3,91 @@ module github.com/giantswarm/capg-firewall-rule-operator
 go 1.18
 
 require (
-	cloud.google.com/go/compute v1.6.0
+	cloud.google.com/go/compute v1.9.0
 	github.com/giantswarm/to v0.4.0
 	github.com/go-logr/logr v1.2.3
 	github.com/google/uuid v1.3.0
-	github.com/googleapis/gax-go/v2 v2.3.0
+	github.com/googleapis/gax-go/v2 v2.5.1
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.5.0
-	github.com/onsi/ginkgo/v2 v2.1.4
-	github.com/onsi/gomega v1.19.0
+	github.com/onsi/ginkgo/v2 v2.1.6
+	github.com/onsi/gomega v1.20.2
 	github.com/pkg/errors v0.9.1
 	go.uber.org/zap v1.19.1
-	google.golang.org/api v0.76.0
-	google.golang.org/genproto v0.0.0-20220414192740-2d67ff6cf2b4
-	k8s.io/api v0.23.6
-	k8s.io/apimachinery v0.23.6
-	k8s.io/client-go v0.23.6
-	sigs.k8s.io/cluster-api v1.0.5
-	sigs.k8s.io/cluster-api-provider-gcp v1.0.2
-	sigs.k8s.io/controller-runtime v0.11.2
+	google.golang.org/api v0.94.0
+	google.golang.org/genproto v0.0.0-20220829175752-36a9c930ecbf
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
```

Comment:
This may be too modern? We are running 1.22 in the MCs

Reply:
They **should** be backwards compatible. I'd say we leave it like that and if it fails on a test installation we can revert to `v0.22.0`

Reply:
Unless there's something specific you think is going to break between these versions

Reply:
Tried it out with the image generated from CI and it seems to work fine